applications/asset_tracker_v2/src/cloud/cloud_codec/nrf_cloud/nrf_cloud_codec.c

```diff
@@ -637,6 +670,67 @@ int cloud_codec_encode_ui_data(struct cloud_codec_data *output,
 	return err;
 }
 
+int cloud_codec_encode_impact_data(struct cloud_codec_data *output,
+			    				   struct cloud_data_impact *imp_buf)
+{
+	int err, len;
+	char *buffer;
+	cJSON *root_obj = NULL;
+
+	if (!imp_buf->queued) {
+		err = -ENODATA;
+		goto exit;
+	}
+
+	root_obj = cJSON_CreateObject();
+	if (root_obj == NULL) {
+		return -ENOMEM;
+	}
+
+	char magnitude[10];
```

Comment:
I think this can be moved to the top of the scope block, it's just easier to see what's being allocated on the stack at a glance. No big deal with this small array, but can be nice to do with larger data structures at least, unless there's an obvious advantage of declaring it close to its first use